src/main/java/Main.java

```diff
@@ -107,8 +106,8 @@ public static void main(String[] args) throws IOException, ClassNotFoundExceptio
         CourseCreationScreen courseCreationScreen = new CourseCreationScreen(courseCreationController, screens, cardLayout);
         screens.add("course", courseCreationScreen);
 
-        StudentMainScreen studentMainScreen = new StudentMainScreen(screens, cardLayout);
-        screens.add("StudentMain", studentMainScreen);
+        StudentMainScreen studentMainScreen = new StudentMainScreen((StudentUser)user, screens, cardLayout);
+        screens.add("main", studentMainScreen);
```

Comment:
If you check the other screens (ex. CalendarScreen), for their "back" or "return" buttons, they should be going back to "StudentMain" screen. You will probably also need to change your own screens so that any "back" buttons go back to "StudentMain"